app/views/groups/_child_groups.html.haml

```diff
@@ -4,7 +4,7 @@
 -#  https://github.com/hitobito/hitobito.
 
 - @sub_groups.each do |label, groups|
-  = section_table(label, groups) do |item|
+  = section_table(label, groups.sort_by(&:sorting_name)) do |item|
```

Comment:
This may be a bit confusing for cases where the short name wouldn't be sorted the same as the full name, since `sorting_name` uses the short name:

![image](https://github.com/hitobito/hitobito/assets/1435346/2bb481c7-eea5-4c82-b61f-731f9a4bfbdd)

So here `SAC Blüemlisalp` has the short name `SAC Z Blmli` and thus appears almost at the end.

Reply:
Also I tried to convert the partial into a view component to avoid to have too much logic in the template. But the most trivial conversion lead to rendering problems which I couldn't solve trivially, since a lot of other partials are involved.